tests/XmlTest.php

```diff
@@ -7,6 +7,11 @@
 
 class XmlTest extends PHPUnit_Framework_TestCase
 {
+    public function testSanitise()
+    {
+        $this->assertEquals('Blah Blah', Xml::sanitise('BlahBlah'));
```

Comment:
Is the blahblah being sanitised containing one of those invisible characters? If so can you make a comment about, otherwise it just looks weird!